tools/lkl/tests/net.sh

```diff
@@ -126,14 +122,38 @@ run_tests()
         lkl_test_exec $script_dir/net-test --backend pipe \
                       --ifname "$fifo1|$fifo2" \
                       --ip $(ip_host) --netmask-len $TEST_IP_NETMASK \
-                      --sleep 30 >/dev/null &
+                      --sleep 1800 >/dev/null &
         cp $script_dir/net-test $script_dir/net-test2
+
+        cat > hijack-test2.conf << EOF
```

Comment:
this is the updated part which doesn't work with FreeBSD.

Reply:
Ah, I see. Another approach could be to try running net-test2 5 times until it succeeds?

Reply:
This is my local trial with the retry you suggested.
I'd like to see if you know how to skip the first failure during this retries: I'd like the run.py to return 0 in this case.

```
% make run-tests tests="--valgrind
 'net.sh -b pipe'"
./tests/run.py --valgrind 'net.sh -b pipe'
  SUITE               net pipe
  TEST       ok       setup_backend pipe
  TEST       ok       nd_create
  TEST       ok       nd_add
  TEST       ok       start_kernel
  TEST       ok       nd_ifindex
  TEST       ok       if_up
  TEST       ok       set_ipv4
  TEST       skip     set_gateway
  TEST       ok       sleep
  TEST       not ok   icmp
pinging 192.168.113.1
recv error (Try again)
  TEST       ok       stop_kernel
  TEST       ok       nd_create
  TEST       ok       nd_add
  TEST       ok       start_kernel
  TEST       ok       nd_ifindex
  TEST       ok       if_up
  TEST       ok       set_ipv4
  TEST       skip     set_gateway
  TEST       ok       sleep
  TEST       ok       icmp
  TEST       ok       stop_kernel
  TEST       ok       cleanup_backend pipe
Summary: 1 suites run, 22 tests, 19 ok, 1 not ok, 2 skipped
make: *** [Makefile:120: run-tests] Error 1
```

Reply:
It won't work as we will get failed tests... Lets go ahead with the sleep, maybe we can find the FreeBSD ping later.

Reply:
ok.  thanks for the review.